keras_core/backend/config.py

```diff
@@ -235,15 +238,19 @@ def standardize_data_format(data_format):
     ]
 )
 def backend():
-    """Publicly accessible method for determining the current backend.
+    """Publicly accessible method for determining the current backend. Note that the
+    backend needs to be set before the import of `keras_core` otherwise the default
+    backend ("tensorflow") will be used if `KERAS_BACKEND` env variable is not set. If
+    you already imported `keras_core` but want to use another backend, make sure to set
+    the `KERAS_BACKEND` env variable or updating the `~/.keras/keras.json` config file
+    before importing `keras_core`.
 
     Returns:
         String, the name of the backend Keras is currently using. One of
             `"tensorflow"`, `"torch"`, or `"jax"`.
 
     Example:
-
-    >>> keras.config.backend()
-    'tensorflow'
+        >>> keras_core.config.backend()
```

Comment:
Unindent.

Reply:
I guess you're following Google's docstring style https://google.github.io/styleguide/pyguide.html, if so, doesn't it need to be indented?